INSTALL.md

```diff
@@ -120,3 +120,17 @@ REM Install "release with debug information" configuration
 cmake --install . --config relwithdebinfo
 REM Well done! Your installed oneTBB is in %TMP%\my_installed_onetbb
 ```
+
+# Installation from vcpkg
```

Comment:
```suggestion
## Installation from vcpkg
```

Reply:
I also recommend placing it before the ``Example of Installation`` section.

Reply:
Thanks for your comment.